fuel-vm/src/interpreter/memory.rs

```diff
@@ -972,11 +972,10 @@ impl OwnershipRegisters {
 
     /// Empty range is owned iff the range.start is owned
     pub(crate) fn has_ownership_heap(&self, range: &Range<Word>) -> bool {
-        // TODO implement fp->hp and (addr, size) validations
-        // fp->hp
-        // it means $hp from the previous context, i.e. what's saved in the
-        // "Saved registers from previous context" of the call frame at
-        // $fp`
```

Comment:
The todo comment here is completely outdated. We do indeed check for this condition below, but for some reason this comment was never removed.